.github/pull_request_template.md

```diff
@@ -1,23 +1,34 @@
 resolves #
+docs dbt-labs/docs.getdbt.com/#
 
 <!---
   Include the number of the issue addressed by this PR above if applicable.
   PRs for code changes without an associated issue *will not be merged*.
   See CONTRIBUTING.md for more information.
+
+  Include the number of the docs issue that was opened for this issue. If
+  this change has no user-facing implications, "N/A" suffices instead. New
+  docs tickets can be created[here](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose).
+-->
+
+### Problem
+
+<!---
+  Describe the problem this PR is solving. What is the application state
+  before this PR is merged?
 -->
 
-### Description
+### Solution
 
 <!---
-  Describe the Pull Request here. Add any references and info to help reviewers
-  understand your changes. Include any tradeoffs you considered.
+  Describe the way this PR solves the above problem. Add as much detail as you
+  can to help reviewers understand your changes. Include any alternatives and
+  tradeoffs you considered.
 -->
 
 ### Checklist
 
 - [ ] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
-- [ ] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
 - [ ] I have run this code in development and it appears to resolve the stated issue
 - [ ] This PR includes tests, or tests are not required/relevant for this PR
-- [ ] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
```

Comment:
Would it be useful to preserve an easy-to-click link somewhere?

https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose

Reply:
Definitely! I have it in the description at the top right now.

Reply:
Ah, I see it in the commented section now 👀

Personally, I'd find it harder to open a docs issue with this version of the PR versus the current one.

But it's totally up to the engineers. I support whichever they find easiest.

Reply:
Can you share what feels harder? I do love bullets for simplicity over sentences in paragraphs, but that's probably not what you mean.

Reply:
I'm gonna guess it's the ability to click a link. I made the top "docs" text a link. Does that help, @dbeatty10?

Reply:
Yep, that is what I meant and that looks easier @nathaniel-may 👍